rotkehlchen/accounting/structures/base.py

```diff
@@ -470,3 +473,15 @@ def serialize(self) -> dict[str, Any]:
 
         balance = abs(self.balance).serialize()
         return {**data, **balance}
+
+
+def get_event_type_identifier(
```

Comment:
you did not do what I asked here. I said hash them too: https://github.com/rotki/rotki/pull/6695#discussion_r1350443642

Since we are no longer interested in this being two-way, by hashing we save a lot of space. This turns into an int.